### ISSUE TITLE
The websocket client throws StackOverflow exception for large messages

### ISSUE THREAD

**Issue by AltTesterBot** [alttester/AltTester-Unity-SDK#821]
## Steps to reproduce

1. Create a new test project: 
   
   ``` 
   $ mkdir new-project 
   $ cd new-project 
   $ dotnet new nunit
   ```

2. Install AltUnityDriver nuget package

   ```
   $ dotnet add package AltUnityDriver --version 1.7.0
   ```

3. Create a simple test that calls `GetScreenshot`. 
4. Run your tests using a high resolution. 

## Actual result

The tests will fail and `GetScreenshot` will raise a `StackOverflow` exception. 

### Additional info

* Depending on the resolution (the size of the message) the test might pass, you need a high resolution to reproduce the issue. 
* The issues it only affects **.NET 6**.

---
<sub>You can find the original issue from GitLab [here](https://gitlab.com/altom/altunity/altunitytester/-/issues/844).</sub>

**Comment by gomboskriszta15** [comment]
Tested this on VuCIty with .NET6 and reproduced it constantly for responses bigger than ~300kB.

**Comment by sandraaltom** [comment]
@gomboskriszta15 Let's see if this bug is still reproducible once https://github.com/alttester/AltTester-Unity-SDK/issues/1111 is implemented.

**Comment by gomboskriszta15** [comment]
After upgrading the websocket library, the exception is no longer thrown in case of .NET6 and .NET7.

**Comment by gomboskriszta15** [comment]
I also tested this with .NET7 using the old version of the websocket and it behaves similarly to the case when I used .NET6 and the old version of the websocket.

**Comment by iulianatodoran** [comment]
Fixed and tested with #1111 

Closing the ticket.
